### DISCUSSION TITLE
FF7FC開発プロジェクトへようこそ

### DISCUSSION THREAD

**Discussion by beta1139** [beta1139/ff7fc-develop#73] [General]
## 👋 ようこそ！
プロジェクトの他のメンバーとつながる場所として Discussions を使用しています

- 疑問に思っていることがあれば質問してください
- 何かアイデアがあれば共有してください
- 他の開発メンバーと交流しましょう
- 私たちは開発に参加したいメンバーを歓迎します
- 一緒にＦＦ７ＦＣを開発しましょう💪

（※コメントするにはgithubアカウントが必要です）

## テスター募集
独自スクリプトでイベント作成が進んでいます
こちらから最新版をダウンロードして動作確認していただき、不具合や感想などの報告をお願いします

## マップ作成者募集
FF7は全部で600程マップが存在しており、現状250程データ化できています
まだ作成できていないマップの8bitドット画像制作をお願いします

**Comment by beta1139** [comment]
@i7-FF7FC @ponyexs
ごめんなさい。ミスって消してしまいました。。🙇

**Reply by beta1139** [reply]
![728](https://user-images.githubusercontent.com/11593576/105627810-63e77180-5e7c-11eb-8faf-1b2926097c59.png)
![729](https://user-images.githubusercontent.com/11593576/105627811-65b13500-5e7c-11eb-88fd-81e6418fd8b2.png)
![730](https://user-images.githubusercontent.com/11593576/105627812-66e26200-5e7c-11eb-88d2-391bfbde18ac.png)
![731](https://user-images.githubusercontent.com/11593576/105627813-66e26200-5e7c-11eb-8de6-66c81e8562b4.png)
![map325](https://user-images.githubusercontent.com/11593576/105627815-677af880-5e7c-11eb-97e8-e0b9b7adaeb2.png)

**Reply by beta1139** [reply]
> @beta1139
> おお、すげえ！ニブル山に背景まである！
> よく作れるよなあ
> 
> @i7-FF7FC
> あざますー既存のチップをコピペして適当にこねくり回してるだけです！
> 空や雲は山の高さを出すために何とか入れたかった感じです。
> 実際にプレイしてると、ルート的にほとんど映らないんですけど😂

**Comment by beta1139** [comment]
> @i7-FF7FC
> お疲れ様です。私もテスト😀
> これアカウント作らないと書けないんですね。
> その辺が若干ハードル高いかもなぁ。
> 
> バグ報告お待ちしてます。
> ～したら落ちた等
> 
> @gamedev222
> そうですね。
> 一応、こちらでアクセス権とか追加しなくても書き込めるもよう
> 
> @i7-FF7FC
> 書き込む際に適当なアカが必要なことは、最初に書いておいた方がいいかも知れませんね。
> でとりあえず様子見しといて、あまりリアクションが無かったらまた考えましょうかw
> たぶん多くの人はGithub自体よく分からないと思いますし、
> 英語だらけのメニュー見てそっ閉じしそうですし😅
> 
> @beta1139
> 一応軽く触れておいた
> 内容が内容だけに大胆に宣伝もしにくいしなあ。。

**Comment by beta1139** [comment]
@i7-FF7FC @ponyexs
ごめんなさい。ミスって消してしまいました。。:bow:

**Comment by beta1139** [comment]
> @ponyexs
> MAP作成や開発のお力には残念ながらなれそうもないので
> テスター代わりに気が付いたことを。
> v4.12aです
> 
> ・ぜんたいかマテリアのAP値が狂うようです。
> 戦闘後APを獲得すると数値がおかしくなります。
> セーブエディタで通常の値（100等）に直しても
> やはり次の戦闘後にはまた数値が狂います
> 
> ・教会屋根裏攻防戦
> 指示「逃げるんだ」を2連続で選んで兵士につかまった瞬間に強制終了
> （「古代種！捕まえたぞ」のメッセージ直後）
> 
> ・列車墓場
> 動かせる列車の入り口に何かの目印があるとありがたいです
> 
> ・7番街の柱のふもと
> 柱の攻防戦開始時、柱へ行かず左側へ進もうとすると
> ティファの警告（クラウド！！早くしなきゃバレットが…）とともに
> クラウドがさらに左側へ移動し、右へ戻れなくなることが一度だけありました
> （直前のデバッグセーブポイントから再開すると次は発生せず）
> 
> それにしても細かいところまで作り込んであって
> ほんと頭が下がります。
> 
> 扉の向こうで待機しているレノとか
> 5番魔晄炉に向かう際の反対側通路の先の特殊戦闘員とのバトルとか！
> 
> 1番魔晄炉破壊後の列車の上のオープニングや
> 主要キャラ登場時の紹介演出など
> 独自要素のセンスも素晴らしいです
> 
> なんといっても豊かな顔芸w

**Reply by beta1139** [reply]
@ponyexs ありがとうございます！確認します！！

**Reply by i7-FF7FC** [reply]
ありがとうございます！
いえいえ、デバッグしていただけるだけでも有難いですよー:smiley:
こういった感想を頂けると何よりのモチベになります！

> ・教会屋根裏攻防戦

スクリプトのパスがミスってたので修正しました。



> ・列車墓場

ここはファミコンリスペクト込みで、敢えて不親切にしているのでちょっと考えますね:sweat_smile:
たしか当初はそれまでのMAP98のように、入れる箇所は地面の色を変えてたんですが、
それだと普通に入っていくだけの作業になっちゃうなぁと思ってて。
一応それっぽいとこいくと、列車動かせる的なヒントは出るようにしてるんですがうーん。

**Reply by i7-FF7FC** [reply]
> ・7番街の柱のふもと

すみません、現時点では分かりませんでした。
ここは以前似たバグがあったので、その時に修正入れたんですが、
「デバッグセーブから再開すると発生しない」というのも併せて考えると、
ひょっとしたら既に直ってる可能性があるかも・・・知れません。
(デバッグセーブから再開すると、最新のファイルが反映されるためです)

ちなみに、直した時期は2019/03(平成最後の更新)みたいなので、
それ以降のアップデータを適用して(ニューゲームかデバッグロードから再開)
同様の現象が確認された場合は、まだ不具合が残っている可能性があります。



ほか、コメントありがとうございます。
レノはなかなか気づいてもらえなかったので感無量です:pray:
ファミコンだと世界観や表現がどうしても分かり辛くなるので、
出来るだけその辺をフォローできればという感じでやっています！

**Reply by ponyexs** [reply]
ご返信ならびに早速の修正ありがとうございます。
早速遊ばせてもらいます

・列車墓場
いわれてみれば確かに確かに。
行けるところにただ順番に入っていくだけになりそうですね。

列車墓場は列車上部の奥まで行けば
ティファのヒントメッセージが表示されるのは知っているのですが、
ただその地点へ行かずとも行き止まりであることが見てわかるため
人によってはそこまで行かず、
結果、メッセージそのものに触れる機会がなく混乱するかも、
という思いもありました。
…近頃の親切なゲームに慣れてしまいファミコンの「味」を忘れてしまっていたようです


・柱のふもと
今回のご報告はすべて4.12aをニューゲームで通して遊んだ場合のものです。
古いバージョンのセーブデータから再開したら
不具合報告の意味がないかもという思いはありましたので。

同じくv4.12aにて列車墓場の通常セーブデータから再開して
ここまで行ってみましたが、今度は発生しませんでした。


そうそう、強制終了するとdbg.txtにその原因が記載されるんですね。
今後強制終了するようなご報告がある場合、
ファイルの内容もなるべく併記するように致します。

**Reply by i7-FF7FC** [reply]
> その地点へ行かずとも行き止まりであることが見てわかるため
> 人によってはそこまで行かず


なるほど、これは言えますね…
これも実は以前もう1パターンあって、MAP99に入った時点で、
最初にヒント出しちゃおうみたいなのもあったんです。
そっちに変えてみようかなぁ・・・今のヒントの場所はそれはそれで残しつつ。

ちょっと後で試してみます！
それ以外に何かアイデアがあればお気軽にどうぞ。
こうやってゲーム内の話が出来るのは楽しいので！


> すべて4.12aをニューゲームで通して遊んだ場合のものです

そうですかーそれは困ったな:sweat:
左に行った際に発生したんですよね？
(覚えていらっしゃらないかも知れませんが・・・)
その際、ティファの台詞中にクラウドが右ではなく左向いてませんでしたかね？
クラウドが更に左に移動してしまう、その前の段階です。




> 強制終了するとdbg.txtにその原因が記載される

あざますー。エラーの種類によっては何も出なかったりするタイプが稀にありますが、
よろしくお願いします:smile:

**Reply by ponyexs** [reply]
向きはそのまま左向きだったかと思います。
直前の行動としてはまずはMAPの下へ行こうとしてティファに止められ、
次に左へ向かったら、という順番でした。
ただ列車墓場からのセーブデータで再開して
同じ行動をとっても再現しないため
何か特殊な条件なのかもしれませんね…
お手を煩わせるような形になってしまい申し訳ありません。

> | それ以外に何かアイデアがあればお気軽にどうぞ。

ありがとうございます。
ではとりあえずは適当な思い付き…といいますかたわ言になりますが…
アイデアの足しになれば幸いです

すごい先の話ですが
ゴールドソーサーでのデートで相手がバレットになった場合
コルネオのイベントで入手した女装グッズが
何らかの形で生かせる機会があると笑いをさそえそう
クラウドちゃん再び！w
とか妄想して遊んでます。

**Reply by i7-FF7FC** [reply]
> 列車墓場

列車墓場のイベントで、
冒頭にヒントを出すようにしてみました。
台詞は本家7rからですが、元々追加してたヒントの一部が
まんまリメイクでも同じだったのでw違和感なしでした。




> ゴールドソーサー

ゴールドソーサーは正直どこまでやれるのかって感じですね:joy_cat:
一応好感度フラグも実装しているので、遊びたいところではありますwこういうのは。
本家リメイクでも見てみたいシーンです。

**Reply by i7-FF7FC** [reply]
> ・柱のふもと

結局原因は不明なので、一応スクリプトの挙動を説明しておきます。
長くなったのでissueの方にトピックを作っているので、
宜しければ参考までにご覧くださいー。

**Reply by ponyexs** [reply]
列車墓場のヒントセリフありがとうございます。
これなら中途半端に迷う人がきっと減るかと思います。

好感度あるんですね！
じゃまずはマリンにお花あげなきゃw

モノを作る力はないので開発そのもののお力にはなれませんが
是非是非最後まで続けてもらえると嬉しいです。

柱の件ですが再度発生したので
状況等はIssueのほうに記載させてもらいます

**Reply by beta1139** [reply]
> ・ぜんたいかマテリアのAP値が狂うようです。
> 戦闘後APを獲得すると数値がおかしくなります。
> セーブエディタで通常の値（100等）に直しても
> やはり次の戦闘後にはまた数値が狂います

@ponyexs 
クラウドの初期装備をぜんたいかに変えて
はじめからゲームを始めて確認してみましたが再現しませんでした
再現方法をもうちょい詳しく教えてください～
https://github.com/beta1139/ff7fc-develop/issues/80

**Comment by ponyexs** [comment]
先ほどのコメントの続きに…とも思いましたが
違う件になりますのでトピックを分けさせてもらいます


崩れる柱からの脱出間際に拾えるラグナロクですが
実際に入手しているはマサムネブレードに
なっているようです
（それにしてもここ、原作通りにただ逃げるだけじゃなく
　アイテム回収要素があるのが楽しいです）


もうひとつ、ひとつ前のバージョンであるv4.12aでの話ですが

神羅ビル61Fにて
カードキー62入手前にエレベータで下に降りると
再度61Fへ行けなくなり進行不能になるようです

もちろん直接61Fは選択できませんが、
一つ下の60Fへエレベータで再度訪れても、そこでティファの
「エアリスを助けなくちゃ」のセリフのため
60F左側の部屋へ入れず、その先の61Fの階段へ戻れなくなるようです
（他に通る方法がある等、私の勘違いだったらすみません）



それとまだ作られていないと思われるレッドの曲を
繋ぎのつもりでとりあえず制作いたしました
といってもNSFは作れないのでMIDIでファミコンぽい感じにした程度ですが…

使いものになりそうでしたら
ホンモノが出るまでの繋ぎにでもご利用いただければ幸いです

https://drive.google.com/file/d/1QaTUdIkV5OBqdqkfJ__9l6aHaJMXdoxa/view?usp=sharing

**Reply by i7-FF7FC** [reply]
> 崩れる柱からの脱出間際に拾える

まさかアレにまでたどり着くとは・・・何というデバッガーの鑑:sparkles:
あれラグナロクじゃなくてラクロナグ(バッタもん)です。
完全に私が思いつきで適当に追加してただけなので、
データ上には無いアイテムなんですよねw
Betaさんに新規に武器追加できるか頼んでみようかな。

> 神羅ビル61F

これは気付かなかった。。
原作調べてみたら、キー62を入手するまで、
61Fのエレベーター開かないようになってましたね…FF7先輩勉強になります。
ホントよく気付いていただけました。感謝です。
同様の処理をしておきました。


> レッドの曲

これ、加入時限定で流れるバージョンですよね。
いやぁ有難いです！とりあえずコスモキャニオンの音楽で代用していたので。

NSFは私も以前覚えようとして煙吹いたので、ファミシンセや8bitplugでやってます。
(最近は専らファミシンセですが)
なので、その辺は大丈夫ですよ。差し替えておきました！

**Reply by ponyexs** [reply]
神羅ビルの件の修正ありがとうございます。

柱のアイテム…
改めて見直してみたら
あらほんとだ、ラクロナグですね
ラグナロクだと思い込んでおりました

そういえばFF5のエクスカリ「パ」ーも、
何年もずぅっとエクスカリバーだと思い込んでいました


マサムネブレードも元のFF7でもネタアイテムですし
アイテムをあえて加えなくても
むしろそのままのほうがニヤリとできるかもしれないですね。


> まさかアレにまでたどり着くとは

FF7FCの高い再現性と追加要素を楽しむために
いろいろ寄り道させてもらってます

バレットへのマテリアの説明を
まさかのネットスラングで返すクラウドさんに笑い、
勝利のポーズのように剣を振り回したり
神羅ビルの通気口で四つん這いになったりするモーションの細かさに唸ったり！




> これ、加入時限定で流れるバージョンですよね。

曲のご採用ありがとうございます。
この曲ってあの場面でしか使われていない曲だったんですね
どおりでFF7オリジナルのほうでコスモキャニオン関連の
イベントをいろいろ進めても一向に耳にしないわけですね

今回の曲にはDMG-CPUというサウンドフォントをとりあえず使いました
どうもファミコンというよりゲームボーイの音に近いらしいですが…

//////////
●ゲーム内のご提案についての回答


・レノ×トンベリ技

兵士を踏んづけた回数は
素晴らしいアイデアだと思います。
何よりこれを花を踏んづけたレノが言うのか！
と突っ込めるところが特に面白いです



・ミッドガル高速（バイク）

ありきたりですが、とりあえずは一つの案として…

追っ手を片付けるのに手間取ると車がガンガン攻撃される、という原作の状況をイメージして
・追っ手との連戦中、時間ごとにHPが減っていき、最後のボス戦が不利な状態で始まる
・一定時間以内に連戦を終わらせないと車が大破
　（ゲームオーバー／またはボスを含む大量の敵との連戦になったり特殊な敵に襲われたり）


------------
そういえば不具合報告はそもそもIssueトピックがありましたね
なるべくそちらに記載させてもらいます。

**Reply by i7-FF7FC** [reply]
> むしろそのままのほうがニヤリとできるかもしれないですね。

そういう記念品扱いもアリっちゃアリですねぇ。どうしようー。



> まさかのネットスラングで返すクラウドさんに笑い、

あれ打った時って、私がこの企画に参加したもうホントに初期の初期の方なんですよ。
だもんで、マテリアの説明の仕方とか全然スクリプトの打ち方が分からず、
適当に誤魔化したのがオチです。
意外と皆さんにウケたので、味を占めてそれ以降もずっと放置してます。


> DMG-CPU

ってvstプラグインか何かですかね？
もしそうなら、自分もバリエーションの一つで入れとこうかなと。

> ミッドガル高速（バイク）

確かに普通に連戦するだけじゃ味が無いので、
何かしらのそういうギミックが追加であれば良さそうですね。
HPが減ったり、レノ等の戦闘中に関するにギミックに関しては、
完全にプログラマーさん案件なので、Betaさんにもぜひ意見聞いてみたいです。

**Reply by beta1139** [reply]
> ミッドガル高速（バイク）

ここは何かしらプログラムでやった方がより良くなるでしょうね
むしろスクリプトだけでよくここまで作ったもんだと感心します

**Reply by i7-FF7FC** [reply]
いえいえ、その都度細かい要望に対応して下さったおかげでございます。
戦闘に関しても、外部スクリプトである程度組めるようになると色々遊べそうです。
敵の基本的なAIに関しては、海外の有志の方が既に解析されているようなので。

バイクはくにおくん？でしったっけ？そんな案もありましたよね。
プログラム的に実装がしやすそうな方をお任せします。

**Reply by ponyexs** [reply]
>ネットスラング

そうだったんですねー　狙っていったのかとw
チュートリアルといえば味方の回復の仕方で実現されていますよね
あれ凄いですね！



> DMG-CPU

これはVSTじゃなくて単なるサウンドフォントというものでして、
MIDIシンセサイザーの、音源部だけのファイルです。
音源の無いスタンドアローン型のソフトMIDIシンセサイザー、
つまり入れ物だけのMIDIシンセがありまして、そこへ音源部として入れて使うためのファイルです。
（私が使っているイレモノは Vurtual MIDI Synth）

VSTみたいな繊細なことはなかなかできないですが、
VSTに対応していない作曲ソフトでも、MIDI OUTポートに
MIDI音源さえ指定できれば使えるので汎用性の高さが便利です

逆に私はファミシンセは知らなかったので
入れさせてもらいました！
ありがとうございます。DMG-CPUよりもファミコンぽい！！


> ここは何かしらプログラムでやった方がより良くなるでしょうね

私はプログラミングはできないので
これがができるということにほとほと感心しますねー
うらやましい！
何せアイデアを実際に形にできるんですから！

**Reply by i7-FF7FC** [reply]
> チュートリアル

そうなんです、一応今はやろうと思えば出来ると思うんですけど、
結構大変だったような...
そもそもこのゲームやろうと思う人が、そこまで基本的なこと
知らないわけないだろうということもあって、放置してるんですけど:sweat_drops:


> DMG-CPU

なるほど、サンプラー的なやつかぁ。
ファミシンセはいい意味で音が細くて好きなんですよね、まさにNES特有のサウンドに近い。

プログラミング出来る方、すごいですよねー。
私もうらやましい限り。

**Comment by ponyexs** [comment]
開発のモチベーションアップに繫がることを願って
柱の攻防戦からカームまでの感想を動画の形で挙げさせてもらいました。
（文字だと感想の前に該当シーンを書く必要があって
　だらだらと長くなりそうな気がしたので。
　とはいえ動画は動画でどちらにしろ視聴のために時間を頂く形になりますね
　申し訳ありません）

最初はリビジョン879のときの動画、途中から880です。


1/3
https://youtu.be/DleFEUR3vDo

2/3
https://youtu.be/-3mwFrIWTCo

3/3
https://youtu.be/vrqGBeeSMNE

**Reply by i7-FF7FC** [reply]
楽しく見させてもらいました！
直近のイベまで網羅してくださり感謝です:bow:

車の駆動音は、ファイル名が「Rain」なのは内緒です！
ピアノイベントは、ちょうど1オクターブ分がファミコンのボタンに収まったので、
妙な使命感に駆られて実装した次第です。
メロディだけではなく、リズムにもある程度判定要素を作っているので、
難しいと思ったんですが、ほんとに良くやっていただきました。すごい:clap:
ティファのあれは、ちょうど限定的にしれーと今度実装しようかと…:flushed:

ところでエルミナの回想シーン、あれビッグス＆ジェシーだったんですね。
エアリスと年齢そんな変わらないと思っていたので、違う人達だと思ってました。。

**Reply by ponyexs** [reply]
ご覧いただきありがとうございました

 >ジェシーとビッグス

単に同じキャラモデルが使いまわされて
いるだけなのかもしれないですけれどね…

アバランチ三人衆の年齢は
私のイメージでは二十代後半～三十代なので、
エルミナの回想のシーンが10年＋αくらいなら
彼らの年齢は十代後半～二十歳前後、と考えれば
本人たちであると考えていいかな、とは思ってます


> ピアノ

おかげでピアノを楽しませてもらいました
伴奏付きなのでリズムをとらなくちゃ、という気になりましたw


それにしても車の音がまさかの雨音！？w

**Reply by i7-FF7FC** [reply]
> ジェシーとビッグス

なるほど、ポリゴンだけだとパッと見分からなかったんですが、
リメイクで見たら、確かにエアリスよりもう一つ上の世代かも知れないとは思いました。

あとビッグスの「バレットいばってんじゃねぇぞ...」みたいな寝言も
バレットに近い年齢と考えた方が辻褄合いやすいですね。
そもそもあの回想シーンは支柱イベントの直後なので、
演出的にもこの二人の方がしっくりはくるかなと、言われてみれば・・・:thinking:

という訳で、余裕ある時にあのシーンもう一度考えてみます！

> 車の音

まぁ一応どっちにでも取れるようにはしてたんですがw
あそこは凄い雨だなという台詞があったので、
土砂降りの叩きつけられる雨音の中、車が進んでいくイメージでした:truck:

**Comment by ponyexs** [comment]
アイデア……
といいますかただの思いつきなので実現の有無は度外視してます。

・スタート時にいるチームチョコボの追加要素をありにしている場合、
神羅ビルのハンドレッドガンナー＆ヘリガンナーの2連戦の
通常バトルBGMがちょっと豪華なVRC6バージョンに（※素材があるなら）

実際のところ、追加要素として
こんな感じに音楽やイラストなど素材が変化する、という方向性はありなんでしょうか？

**Reply by ponyexs** [reply]
[1-10-Fighting_vrc6_2.zip](https://github.com/beta1139/ff7fc-develop/files/5939318/1-10-Fighting_vrc6_2.zip)
とりあえず先にVRC6もどきを。需要の有無は置いといて。

**Reply by i7-FF7FC** [reply]
ありがとうございます＆お疲れ様です！

> こんな感じに音楽やイラストなど素材が変化する、という方向性はありなんでしょうか？

基本ファミコンの仕様内ならい良いんではないかと。
このVRC6って拡張音源的なやつですよね？
まいったな…こういうのはよく分からなくて判断できないw
こういう時人数多いと、適当に判断仰げるんだけどどーしよう:sweat_smile:

**Reply by ponyexs** [reply]
用意した上のもはVRC6「もどき」なので現実のVRC6よりもやや同時発音数が多いです

それはともかくとしまして、
そもそも拡張チップの要素を入れるかどうかは必ず好き嫌いがあるはずなので
もっと多くの方の意見が欲しいですよね…

純粋なファミコンだけの要素で再現してこそ、
これぞ俺たちの待ち望んだファミコン版！という思いをもつ人も多そうな気もしますし。

思ったのですが、このGithubサイトは検索でたどり着くまでが難しいかもしれません。

最近注目されていると思われる、
コメントが集中しているYoutubeのFF7FC動画サイトですが
あちらを見てこのゲームを探そうとすると、動画のタイトルなどから
　FF7 ファミコン
　ファイナルファンタジー7FC
あたりのキーワードで探してしまうんですよね
ですがこれですとGithubサイトがGoogleで引っかからないようです

なのでこのサイト自体にたどり着ける人自体が少ないのかもしれませんね…

**Reply by i7-FF7FC** [reply]
githubは真面目に話し合ったり、平和なコミュニティ形成()にはすごく良い場所ですが、
導線的にはちと弱いですよね。
youtubeに入り口はあった方が良いと思うのは私も同感です。
今やってるイベが一通り片付いたら、ちょっと考えてみようかなとは一応思ってるんですが。

ところでbetaさん、このディスカッションのソート順って、Newestをデフォに出来ないですかね？
レスが増えるほどめっちゃ埋もれるので。

**Reply by beta1139** [reply]
デフォルトNewestはちょっと分かりませんでした
ここは新しいメンバー専用の Discussion にして、
個別の話題は新しく IssueなりDiscussion なり作った方がよいのかも

**Reply by i7-FF7FC** [reply]
なるほど！そうしますか。
画像投下等は今後分けてみようかと思います。

**Reply by ponyexs** [reply]
承知ですー

**Comment by ponyexs** [comment]
[mortorball_substitute.zip](https://github.com/beta1139/ff7fc-develop/files/5939273/mortorball_substitute.zip)

[素材：モーターボール・当面の代替品…のつもり]

これディスカッションのトピックでいいのかな…

正直ファミコン風のドット絵ではありませんが
画像無しよりはいいかもと思いとりあえず用意しました

使えそうなら加工等してご利用ください

**Reply by i7-FF7FC** [reply]
引き続きありがとうございます！
モーターボールは、昔作りかけたまま放置してたのが実はあるんですよ。
こんなのw
![motor_ball](https://user-images.githubusercontent.com/11593576/107217710-e746bc80-6a51-11eb-9d29-6314650645c2.png)
![motor_ball2](https://user-images.githubusercontent.com/11593576/107217715-e9a91680-6a51-11eb-9cf0-149f7e3d5963.png)

**Reply by ponyexs** [reply]
こりゃ凄い！　作りかけどころかほぼ完成形に見えます！　　私が用意したのは無かったことにしてください

**Reply by i7-FF7FC** [reply]
これ確か中央の本体？みたいなのがうまく出来なくて、
いっぱい砲台置いたり、警備兵乗せてお茶を濁しそうとして、
よく分からなくなって放置してた状態だったと思いますw

ちなみに画像作成は、「EDGE」っていう256色専用のドットエディタ(Ver1の方)を
メインで使用しているので、素材を提供して下さる場合は、入れておくと便利ですよ！:smiley:
例えば「map_chip.bmp」をエッジで開くと、
カラーパレットに今使っているVirtua NESパレットが読み込まれているので、
まずそれを適当な名前で保存して、以後新たな画像はそのパレットの範囲内で調整する感じです。

**Reply by ponyexs** [reply]
なるほど、そういった感じで色を指定して使うんですね
ありがとうございます。
早速EDGE1の導入とパレット設定を行いました。
こうやって見るとファミコンって結構カラフルですねえ

さて、あとは私がドット絵が描けるようになるだけ…！（困

**Reply by i7-FF7FC** [reply]
> 結構カラフルですねえ

本当ですか？私は逆に初めて触ったときコレしか使えないのかよーとショッックでしたw
ちなみに実際のファミコンはRGB値で管理してないようなので、
このvirtuaNESパレットはあくまで疑似的なやつのようです。

ドットに関しては、最初はキャラチップ(16x16)とか、
サイズの小さいものから始めてみるのも良いかも知れません。
(しれーと制作陣にスカウトしていくスタイル:smirk:)

**Reply by ponyexs** [reply]
せっかくの素晴らしい作品に猛烈に足を引っ張りそうな気がするので
当面はテストプレイへ出歯亀いたす所存でございまする m(_ _)m

…と逃げつつ、一部のOGGはループ位置が気になるものがありましたので
もしかしたらそこだけ手を出す、かも…（余計なことでしたら済みません）

**Reply by i7-FF7FC** [reply]
とんでもない、テストしていただけるだけでも助かっています:pray:
自分も最初はデバッガーから始めて、出来そうなことを少しづつやってたら、
徐々に欲も出てカバー範囲が増えていった感じなので、気楽にトライしていただければ幸いです:ok_hand:

**Reply by ponyexs** [reply]
ありがとうございます。
とりあえず出来そうなことからお手伝いさせてもらいますー

**Comment by i7-FF7FC** [comment]
マップ投下
ニブル魔晄炉内
![327](https://user-images.githubusercontent.com/11593576/107215249-69cd7d00-6a4e-11eb-8098-80e04ee07e28.png)
![328](https://user-images.githubusercontent.com/11593576/107215257-6c2fd700-6a4e-11eb-9975-2566541cc93e.png)

**Reply by ponyexs** [reply]
凄い！　あのシーンだ！　端のポットが暗くなっているところがリアルですね

**Reply by i7-FF7FC** [reply]
あざます！
元が3Dマップなので、影の部分は結構大事にしてます:smiley:

しかしながら、まだ色々用意しないといけなそうなのがあって、
どう誤魔化すか、絶賛頓挫中です。

**Reply by beta1139** [reply]
すげえなあ

**Reply by i7-FF7FC** [reply]
自分としては、パレット3で茶色削って水色を組み込めたのが良かったです。
寒色のアクセントが効いて、実験室っぽい雰囲気が出たので。

**Comment by sirasu2657** [comment]
はじめまして。
気になる企画だったのでコメントさせていただきます。

私はグラフィック方面に心得があるのですが、なにかお役に立てることはありますか？
マップ制作募集とのことですので、未制作リストなどがあれば制作いたします。
